mobile/evidence/ios/Leaf/Sources/Leaf/Components/LeafText.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  LeafText.swift
+//  
+//
+//  Created by Cris Messias on 06/10/23.
+//
+
+import SwiftUI
+
+//MARK: - TitleModifier
+
+public struct TitleModifier: ViewModifier {
+    @Environment(\.leafTheme) private var theme
+    public func body(content: Content) -> some View {
+        content
+            .font(theme.font.titleLarge)
+            .bold()
+            .lineSpacing(0.2)
+            .foregroundStyle(theme.color.content.primary)
+
+    }
+}
+public extension View {
```

Comment:
I've seen this pattern in the swiftui library. However, since we are making extensions for Text, what do you think of making a Text extension instead of a View extension?
```suggestion
public extension Text {
```

Reply:
Amazin! Is better. The code is more clean.